src/hooks/useArrowKeyNavigation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'useArrowKeyNavigation'
+type: 'hook'
+categories:
+- Hooks
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: ''
+---
+
+`useArrowKeyNavigation` adds arrow key navigation through any component.
+
+## Basic Usage
+To enable item navigation using the arrow keyboard specifies the CSS selectors string that indicates to which elements
+the user can navigate.
+
+```jsx live
+() => {
+  const parentRef = useArrowKeyNavigation({
+    selectors: 'a:not(:disabled),button:not(:disabled),input:not(:disabled)',
+  });
+
+  return (
+    <div ref={parentRef}>
+      <Button className="mb-2 mb-sm-0" autoFocus>Brand</Button>{' '}
+      <Button className="mb-2 mb-sm-0" variant="outline-brand">Outline Brand</Button>{' '}
+      <Button className="mb-2 mb-sm-0" variant="outline-primary">Outline Primary</Button>{' '}
+      <Button className="mb-2 mb-sm-0" variant="tertiary">Tertiary</Button>{' '}
+    </div>
+  );
+};
+
+```
+
+## Ignored arrow keys
+To disable listening for keystrokes, use `ignoreKeys` to add ignored keys to the array.
```

Comment:
It might be helpful to describe the intended behavior for the component example here so consumers don't necessarily need to look at the code to understand what the preview is showing regarding the ignored arrow keys.

Reply:
Added, thanks